tests/foreman/cli/test_import.py

```diff
@@ -222,18 +259,47 @@ def gen_import_repo_data():
 
 def gen_import_cv_data():
     """Random data for Content View Import tests"""
+    return ({
+        u'users': [{
+            u'key': 'organization_id',
+            u'key_id': type(u'')(i + 1),
+            u'organization': gen_string('alphanumeric')}
+            for i in range(3)
+        ],
+        u'content-views': [{
+            u'key': u'org_id',
+            u'key_id': type(u'')(i + 1),
+            u'channel_name': gen_string('alphanumeric'),
+            u'channel_label': gen_string('alphanumeric')}
+            for i in range(3)
+        ]},
+    )
 
+
+def gen_import_rh_repo_data():
+    """Random data for Organization Import tests"""
+    org_ids = [type(u'')(org_id) for org_id in sample(range(1, 1000), 3)]
+    # wipe all channel names and labels excepting channel id 106
     return (
         {
             u'users': [{
+                u'key': u'organization_id',
                 u'key_id': type(u'')(i + 1),
-                u'organization': gen_string('alphanumeric')
+                u'organization_id': org_ids[i],
+                u'organization': gen_string('alphanumeric'),
             } for i in range(3)],
-            u'content-views': [{
-                u'key_id': type(u'')(i + 1),
+            u'channels': [{
+                u'key': u'channel_id',
+                u'key_id': type(u'')(i),
+                u'channel_label': u'',
                 u'channel_name': gen_string('alphanumeric'),
-                u'channel_label': gen_string('alphanumeric'),
-            } for i in range(3)]
+            } for i in set(range(101, 113)) - set(range(106, 107))] + [
```

Comment:
This piece will change name of all repos excepting 107 (rhn-tools-rhel-x86_64-server-7). This will make hammer to skip them and to only recognize and synchronize a single RH repo in order to speed up the test.
rhn-tools-rhel-x86_64-server-7 is the smallest one